src/Core/components/BezierControls.js

```diff
@@ -124,6 +124,7 @@ export default class BezierControls extends InlineComponent {
       isArray: true,
       entryPrefixes: ["direction"],
       hasEssential: true,
+      isLocation: true,
```

Comment:
I don't necessarily have a better idea in mind, but it feels like this attribute could be easy to forget when defining new state variables. I also am not sure what I should do to try to audit that you have added this in all of the right places so far.

Reply:
Added a note explaining fixedLocation where the state variable is defined in BaseComponent.